gradle/libs.versions.toml

```
@@ -1,5 +1,5 @@
 [versions]
-protocol = "3.0.0.Beta6-20250506.012145-17"
+protocol = "3.0.0.Beta7-SNAPSHOT"
```

Can you pin the version
```suggestion
protocol = "3.0.0.Beta7-20250619.102015-10"
```

---

Done

---

See my other comment